recipes/cpr/all/conanfile.py

```diff
@@ -67,6 +64,8 @@ def _configure_cmake(self):
         return cmake
 
     def build(self):
+        if self.options["libcurl"].with_ssl != "openssl":
+            raise ConanInvalidConfiguration("CPR requires libcurl built with the option with_ssl=\"openssl\".")
```

Comment:
This might be nitpicking but I would keep it lower case as the recipe itself is lower cased 😄 
```suggestion
            raise ConanInvalidConfiguration("cpr requires libcurl built with the option with_ssl=\"openssl\".")
```